### PR TITLE
docs: create page for typography in docs section for patternflyelements.org

### PR DESCRIPTION
**Pull request by starryeyez024**
## Add page within docs site for typography

Borrowed some of the text from the readme files from pfe-styles and pfe-sass so that there would be evidence of typography styles in the docs.

### Preview

Link(s) to demo page(s) where this element can be viewed:
- [Link](https://deploy-preview-1753--patternfly-elements.netlify.app/theming/typography/) 
 

### Ready-for-merge Checklist

- [x] Documentation (README.md, WHY.md, etc.) updated or added. 

### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**

### CODE REVIEW

File: docs/_data/todos.json [hunk 1]
```diff
@@ -1 +1 @@
...
... will be removed in 2.0"},{"file":"elements/pfelement/src/pfelement.js","tag":"TODO","line":60,"ref":"","text":"Add config validation in the future."},{"file":"elements/pfelement/src/pfelement.js","tag":"TODO","line":174,"ref":"","text":"Deprecated with 1.0"},{"file":"elements/pfelement/src/pfelement.js","tag":"TODO","line":378,"ref":"","text":"Deprecated for 1.0 release"},{"file":"elements/pfelement/src/pfelement.js","tag":"TODO","line":381,"ref":"","text":"Migrate this out of schema for 1.0"},{"file":"elements/pfelement/src/pfelement.js","tag":"TODO","line":603,"ref":"","text":"maybe we should use just the attribute instead of the class?"},{"file":"elements/pfelement/src/pfelement.js","tag":"TODO","line":634,"ref":"","text":"--theme will be deprecated in 2.0"},{"file":"elements/pfelement/src/pfelement.js","tag":"TODO","line":893,"ref":"","text":"add support for a validation function"},{"file":"elements/pfelement/src/pfelement.js","tag":"TODO","line":1043,"ref":"","text":"Should this error be reported to the console?"},{"file":"elements/pfelement/src/pfelement.js","tag":"TODO","line":1086,"ref":"","text":"add a warning when a component property conflicts with a global property."}]
```
Comment (starryeyez024) [inline docs/_data/todos.json:1]: @kylebuch8  is it normal for this to appear in the diff like this?

Comment (heyMP) [reply docs/_data/todos.json:1]: It makes sense if this is generates a list of todos.  I've not seen it before though 🤷

File: docs/theming/typography.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+::: section  
+## General
+
+The pfe-styles component ships with a variety of stylesheets to allow developers to opt-into what they need, and leave behind what they don't. We leverage naming conventions that match up with the [core PatternFly project](https://github.com/patternfly/patternfly), and we follow best practices from the web and accessibility. 
+
+### pfe-base.css 
+
+- This stylesheet provides normalize styles and styles for standard typographical HTML tags such as `<ul>`, `<h2>`, etc. 
+- The typographical elements are opt-in, in that you must use the wrapper class of `.pfe-c-content` around them for them to be applied.
+- The "normalize" settings are very lightweight and help clear common issues with default browser styles such as collapsing borders on tables or clearing padding and margins on `ul` tags.
```
Comment (heyMP) [inline docs/theming/typography.md:24]:
```suggestion
- The "normalize" settings are very lightweight and help clear up common issues with default browser styles such as collapsing borders on tables or clearing padding and margins on `ul` tags.
```

File: docs/theming/typography.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+- We recommend loading this as a standard stylesheet in your project
+
+### pfe-typography-classes.css
+
+- This stylesheet includes modifier clases you may use with any markup to invoke those particular heading styles. Please see the [documentation](https://ux.redhat.com/foundations/typography) for additional details on styles. This is very useful if you do not already have typographical classes defined.
```
Comment (heyMP) [inline docs/theming/typography.md:29]:
```suggestion
- This stylesheet includes modifier classes you may use with any markup to invoke those particular heading styles. Please see the [documentation](https://ux.redhat.com/foundations/typography) for additional details on styles. This is very useful if you do not already have typographical classes defined. 
```

File: docs/theming/typography.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+<h2 class="pfe-title--md">Hello</h2>
+<h2 class="pfe-title--sm">Hello</h2>
+<h2 class="pfe-title--xs">Hello</h2>
+<p class="pfe-text--lg">World</p>
+<p class="pfe-text--md">World</p>
+<p class="pfe-text--sm">World</p>
+<p class="pfe-text--xs">World</p>
+```
+ 
+<pfe-cta >
```
Comment (heyMP) [inline docs/theming/typography.md:56]:
```suggestion
<pfe-cta>
```

Comment (heyMP) [reply docs/theming/typography.md:56]: Not sure if this is a typo or not?

**Review by heyMP — COMMENTED** [review]
Looks great @starryeyez024!

File: docs/theming/typography.md (hunk 4 above)
Comment (starryeyez024) [reply docs/theming/typography.md:56]: umm.. yes let's call it a typo :) good catch

**Comment by starryeyez024** [comment]
thank you @heyMP! :D

**Review by heyMP — APPROVED** [review]
Lots Gooder Typography Mate!

**Pull request merged**